src/app/shared/components/favorite-mark/favorite-mark.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+import { AuthPopUpService } from '../auth-pop-up/auth-pop-up-service/auth-pop-up.service';
+
+@Component({
+  selector: 'app-favorite-mark',
+  templateUrl: './favorite-mark.component.html',
+  styleUrls: ['./favorite-mark.component.scss']
+})
+export class FavoriteMarkComponent {
+  //change after login system is implemented
+  readonly isFavorite = false;
```

Comment:
readonly isFavorite = false;
it have no sense because you won't be able to change it on true (readonly)

Reply:
it a mock data. It's gonna be changes once the authentication is implemented